config.ini

```diff
@@ -15,7 +15,7 @@ database = NONE
 
 [OPA]
 opa =  true
-opaexe = $OPA_HOME/opa
+opaexe = /usr/bin/opa
```

Comment:
Why this is changed?

Reply:
it was changed to run in my local
returned to the previous state